src/module.c

```diff
@@ -5146,7 +5146,8 @@ robj *moduleTypeDupOrReply(client *c, robj *fromkey, robj *tokey, int todb, robj
  *   pointer if the top-level value pointer is defragmented and consequently changes.
  *
  * * **mem_usage2**: Similar to `mem_usage`, but provides the `RedisModuleKeyOptCtx` parameter 
- *   so that meta information such as key name and db id can be obtained.
+ *   so that meta information such as key name and db id can be obtained, and the `sample_size` for size estimation
+ *   (see `memoryCommand` and `objectComputeSize` for more details)
```

Comment:
These comments are used to generate the online documentation: https://redis.io/topics/modules-api-ref
we can refer to the commands documentation, but not the redis source code.
```suggestion
================================================================================
 *   so that meta information such as key name and db id can be obtained, and
 *   the`sample_size` for size estimation (see MEMORY USAGE command).
```